pypeit/calibrations.py

```diff
@@ -334,15 +349,17 @@ def get_align(self):
             self.alignments.is_synced(self.slits)
             return self.alignments
 
-        msalign = buildimage.buildimage_fromlist(self.spectrograph, self.det, self.par['alignframe'], align_files,
+        msalign = buildimage.buildimage_fromlist(self.spectrograph, self.det,
```

Comment:
Why isn't dark=self.msdark being passed here as above for get_tiltimg?

Reply:
Good catch.  I'm passing it now.